lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -144,34 +144,34 @@ def setUp(self):
             display_name="Group Project2"
         )
 
-        self.course_content = ItemFactory.create(
-            category="videosequence",
-            parent_location=self.chapter.location,
-            data=self.test_data,
-            display_name="Video_Sequence",
-        )
-
         self.course_content2 = ItemFactory.create(
             category="sequential",
             parent_location=self.chapter.location,
             data=self.test_data,
             display_name="Sequential",
         )
 
-        self.content_child = ItemFactory.create(
-            category="video",
-            parent_location=self.course_content.location,
-            data=self.test_data,
-            display_name="Video"
-        )
-
         self.content_child2 = ItemFactory.create(
             category="vertical",
             parent_location=self.course_content2.location,
             data=self.test_data,
             display_name="Vertical Sequence"
         )
 
+        self.course_content = ItemFactory.create(
```

Comment:
Nit: It's fine to relocate these factory calls, but the code structure now seems odd by having the secondaries above the primaries